extruct/rdfa.py

```diff
@@ -26,6 +26,24 @@
     "fb": "http://ogp.me/ns/fb#"
 })
```

Comment:
@ShivinDass do you think a fix similar to above could be possible here?

Reply:
> @ShivinDass do you think a fix similar to above could be possible here?

@lopuhin I took a look at rdflib in more detail and updating the initial_context is a more efficient implementation. I've made the changes and have updated the pull request.